srcpkgs/hunspell-sv_SE/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'hunspell-sv_SE'
+pkgname=hunspell-sv_SE
+version=2.41
+revision=1
+noarch=yes
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Swedish sv_SE dictionary for hunspell"
+maintainer="Venca Uruba <uruba@outlook.com>"
+license="GNU-GPL-v3, GNU-GPL-v2, LGPL-v3+"
+homepage="https://extensions.libreoffice.org/extensions/swedish-spelling-dictionary-den-stora-svenska-ordlistan/"
+distfiles="https://EXtensions.libreoffice.org/extensions/swedish-spelling-dictionary-den-stora-svenska-ordlistan/$version/@@download/file/ooo_swedish_dict_$version.oxt>dict-da-$version.zip"
+checksum=48c8d0d548232215384c47ba18ac03bbef3af4c11ffd5e1057272353ba60225f
+
+do_install() {
+	vinstall dictionaries/sv_SE.aff 644 usr/share/hunspell
+	vinstall dictionaries/sv_SE.dic 644 usr/share/hunspell
+	vinstall dictionaries/sv_FI.aff 644 usr/share/hunspell
+	vinstall dictionaries/sv_FI.dic 644 usr/share/hunspell
+	vlicense LICENSE_en_US.txt
```

Comment:
GNU GPL licenses are in the system under /usr/share/licenses, you do not need to use vlicense to install them, vlicense is used for licenses like MIT and ISC that have copyright headers that need to be included with the package.